app/Http/Controllers/Admin/GalleryController.php

```
@@ -20,36 +23,69 @@ public function index()
-            'title' => $request->input('title:eo'),
-            'description' => $request->input('description:eo'),
-        ], 'eo');
-
-        $gallery->setTranslations([
-            'title' => $request->input('title:ru'),
-            'description' => $request->input('description:ru'),
-        ], 'ru');
+        $validated = $request->validated();
+
+        // Handle cover image upload using PortfolioImageService
+        $coverImagePath = null;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$coverImagePath'. (undefined)

(UnusedLocalVariable)